core/templates/github/docker/lint.yml

```diff
@@ -1,16 +1,17 @@
-name: Docker Image CI
-
+name: Docker file Lint
 on:
   pull_request:
     paths:
-      - "Dockerfile"
-
+      - "**Dockerfile"
 jobs:
   lint:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v2
-      - uses: hadolint/hadolint-action@v1.5.0
-        with:
-          dockerfile: Dockerfile
+      - run: |
+          for dockerfile in $(git ls-files **/*Dockerfile* Dockerfile)
+          do
+            echo "Starting lint for the Dockerfile $dockerfile"
+            docker run --rm -i hadolint/hadolint < $dockerfile
+          done
```

Comment:
Hadolint can lint multiple Dockerfiles with one command.
Check this issue comment for an example:
https://github.com/hadolint/hadolint-action/issues/3#issuecomment-912346786

Reply:
I had tested it this way, but it couldn't match the Dockerfile at the root of the file
https://github.com/pipelinit/pipelinit-sample-docker/runs/3704330499?check_suite_focus=true

I tried it again here and now using the 'find' command to match the Dockerfiles, and it worked just fine
can you look again?

This is the run 
https://github.com/pipelinit/pipelinit-sample-docker/runs/3725316341?check_suite_focus=true